src/components/SearchBar/SearchBar.js

```diff
@@ -3,43 +3,51 @@ import { gql, useLazyQuery } from '@apollo/client';
 import MaterialsDropdown from '../MaterialsDropdown/MaterialsDropdown';
 import './SearchBar.css';
 
-const SearchBar = ({ updatePlaceIds }) => {
+const SearchBar = ({ updateLocationDetails }) => {
   const [materialId, setMaterialId] = useState('');
   const [location, setLocation] = useState('');
 
   const updateMaterialId = (id) => setMaterialId(id);
 
-  const GET_PLACE_IDS = gql`
+  const GET_LOCATION_DETAILS = gql`
```

Comment:
I am glad we were able to pivot and collaborate with the back-end to fix the placeId issue.